packages/commonwealth/client/scripts/views/pages/discussions/CommentCard/CommentCard.tsx

```diff
@@ -14,8 +14,11 @@ import {
   ViewUpvotesDrawerTrigger,
 } from 'client/scripts/views/components/UpvoteDrawer';
 import clsx from 'clsx';
+import { useFlag } from 'hooks/useFlag';
 import type Comment from 'models/Comment';
 import { useFetchConfigurationQuery } from 'state/api/configuration';
+import { useCreateCommentSubscriptionMutation } from 'state/api/trpc/subscription/useCreateCommentSubscriptionMutation';
```

Comment:
can you add index file to subscription directory so that we can import

import { useCreateCommentSubscriptionMutation, useDeleteCommentSubscriptionMutation } from 'state/api/trpc/subscription/'

Reply:
Can we discuss + implement this in another PR? 

I don't necessarily think we should do this (willing to be wrong) but I'd also like to stop increasing the scope of PRs like this.

If we implement my proposal to migrate away from squash merges we can send in smaller PRs like this and stack them and reduce PR merge latency.

Reply:
I created an issue for this but we can't implement it in this PR as we're already in a rush to get this merged: 

https://github.com/hicommonwealth/commonwealth/issues/8712